src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
 public class Duke {
```

Comment:
Please add header comment for each class and non-trivial method if you have time in the near future